assets/css/vendor/filepond/filepond-preview.css

```diff
@@ -0,0 +1,163 @@
+/*!
```

Comment:
Is it possible to add custom styling to the image preview to make the green overlay blue?

![image](https://user-images.githubusercontent.com/1770201/110256501-2acd1180-7f91-11eb-9e46-6483e9f63a3b.png)

Reply:
@polevaultweb 
I've committed new changes for this.